root/static/less/font-awesome/_variables.less

```diff
@@ -3,7 +3,7 @@
 
 @fa-font-path:         "/static/fonts";
 @fa-font-size-base:    16px;
-@fa-font-display:      auto;
+@fa-font-display:      swap;
```

Comment:
I don't think this is appropriate. For normal text, swap makes sense because it can be rendered with a fallback font. Font-awesome is kind of a hack, and using custom characters to render icons. There is no fallback font that would ever make sense to use.

Reply:
Good call, removed. Is there any way to use `font-display: swap` for the actual text, though? Lighthouse is right that the text jumps in about 1500ms after request starts.

Reply:
We aren't using a custom font for the actual text, so I'm not sure what the purpose of font-display: swap would be.

Reply:
Makes sense! Resolving this.